packages/react/jest.config.js

```diff
@@ -1,4 +1,14 @@
+const commonConfig = require('../../build/jest/jest.config.common')
+
 module.exports = {
-  ...require('../../build/jest/jest.config.common'),
+  ...commonConfig,
   name: 'react',
+  moduleNameMapper: {
+    ...commonConfig.moduleNameMapper,
+    'docs/(.*)$': `<rootDir>/../../docs/$1`,
+
+    // Legacy aliases, they should not be used in new tests
+    'src/(.*)$': `<rootDir>/src/$1`,
+    'test/(.*)$': `<rootDir>/test/$1`,
+  },
```

Comment:
These mapping is specific for `@stardust-ui/react` and should not be used in new packages.